src/content/docs/en/guides/images.mdx

```diff
@@ -657,7 +657,33 @@ If you were using the image integration in Astro v2.x, complete the following st
     })
     ```
 
-2. Migrate any existing `<Image />` components.
+2. Update types
+
+  In some projects, `@astrojs/image` had special types set up. If you have these, you need to change them back to the default Astro types.
+  
+  You need to update the `src/env.d.ts` file if you have one.
+
+  ```ts
+  // src/env.d.ts
+  - /// <reference types="@astrojs/image/client" />
+  + /// <reference types="astro/client" />
+  ```
+
+  Alternatively, if you have types set in your `tsconfig.json`, update it like so.
+
+  ```json
+  // tsconfig.json
+  {
+    "compilerOptions": {
+      - "types": ["@astrojs/image/client"]
+      + "types": ["astro/client"]
+    }
+  }
+  ```
+
+  If you can't find `@astrojs/image/client` in either of these files, don't worry: not everyone has them. You can safely continue.
```

Comment:
``````suggestion
2. Update types (if required).

  If you had special types configured for `@astrojs/image` in `src/env.d.ts`, you may need to change them back to the default Astro types if your upgrade to v3 did not complete this step for you.

  ```ts title="drc/env.d.ts" del={1} ins={2}
  /// <reference types="@astrojs/image/client" />
  /// <reference types="astro/client" />
  ```

  Similarly, update `tsconfig.json` if necessary:

  ```json title="tsconfig.json" del={3} ins={4}
  {
    "compilerOptions": {
      "types": ["@astrojs/image/client"]
      "types": ["astro/client"]
    }
  }
  ```
``````